Common/AR/source/ArcGISArViewInterface.cpp

```diff
@@ -73,11 +72,17 @@ ArcGISArViewInterface::ArcGISArViewInterface(bool renderVideoFeed, bool tryUsing
     {
       case Qt::ApplicationSuspended:
         if (m_tracking)
```

Comment:
Should `m_tracking` be `m_trackingEnabled`?

`m_tracking` and `m_trackingPaused` seem too similar.